quickfixj-base/src/main/java/quickfix/MessageUtils.java

```diff
@@ -118,79 +117,6 @@ public static Message parse(MessageFactory messageFactory, DataDictionary dataDi
         return message;
     }
 
-    /**
```

Comment:
This removed stuff went to `MessageSessionUtils`, right?

Reply:
Yes , confirmed